readme.md

```diff
@@ -136,6 +136,14 @@ The attribute should be an image URL:
 
 Read more about the `media` parameter in the [in the Pinterest documentation](https://developers.pinterest.com/docs/widgets/pin-it/#source-settings).
 
+### Telegram
+
+You can set `data-text` attribute to set a text of the message.
+
+```html
+<div class="telegram" data-text="Hello, world!">Send</div>
```

Comment:
Good point. As we have ilyabirman.com as an example, I think it should be «Ilya Birman's website: design changes the world!»